src/lib/buy/index.tsx

```diff
@@ -525,6 +529,57 @@ function BuyOrder(props: BuyOrderProps) {
               <Text>- ({order.status})</Text>
             </Box>
           )}
+
+          {order &&
+            order.status === "filled" &&
+            (order as Awaited<ReturnType<typeof getOrder>>) &&
```

Comment:
without this the type checker cant' see 'order.quantity' or order.start_at and order.end_at.